client/src/components/useDarkMode.test.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import useDarkMode from "../hooks/useDarkMode";
+import { Nav } from "./navBar";
+import { renderIntoDocument, cleanup } from "@testing-library/react";
+import Enzyme, { mount } from "enzyme";
+
```

Comment:
really would have liked to see more tests one is not really enough three or four would have been better